Mail2Bug/WorkItemManagement/SimpleWorkItem.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+
+namespace Mail2Bug.WorkItemManagement
+{
+    public class SimpleWorkItem
+    {
+        /// <summary>
+        /// The work item ID.
+        /// </summary>
+        public int Id { get; set; }
+
+        /// <summary>
+        /// Work item title.
+        /// </summary>
+        public string Title { get; set; }

```

Comment:
This, in fact, was the reason I chose to add those fields as members. These are standard fields (ID, Title, Owner, State) that should be part of any work item management system but they might be named differently. So abstracting them under these first class members should help resolve any issues due to the difference in terminology. For example, without this, the default email template may not work for anything other than TFS since the fields have a different name and we are using a specific name in the template.